app/src/main/java/com/powerhungers/fomezero/feature/login/data/remote/model/LoginResponse.kt

```diff
@@ -21,4 +21,8 @@ data class LoginResponse(
     val createdAt: String?,
     @SerializedName("updatedAt")
     val updatedAt: String?,
+    @SerializedName("token")
+    val token:String?,
+    @SerializedName("refresh_token")
+    val refresh_token:String?
```

Comment:
```suggestion
    val refreshToken:String?
```
o padrão de variaveis no android é camelCase e não snake_case como vem do backend